addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -0,0 +1,208 @@
+/** @odoo-module **/
+
+import options from 'web_editor.snippets.options';
+import {websiteDomain} from 'website.utils';
+
+options.registry.SocialMedia = options.Class.extend({
+    /**
+     * @override
+     */
+    async willStart() {
+        this._super(...arguments);
+        const websiteId = websiteDomain(this)[2][2];
+        const target = this.$target[0];
```

Comment:
seems useless as variable only used once, especially since in renderListItems you don't do it but reuse this.$target[0] multiple times